routes/middleware/post_place.js

```diff
@@ -1,7 +1,8 @@
 const request = require('request');
+require('dotenv').config();
 
 module.exports = (req, res, next) => {
```

Comment:
This middleware should only run if the user hasn't selected a place that already exists